db/migrate/20230607063136_create_comments.rb

```diff
@@ -0,0 +1,15 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
```

Comment:
- [ ] As suggested by the previous reviewer that it would be nice if you have the author_id column in the comments table to follow the project requirements 👍

Kindly verify if you are using the latest requirements document which is version v1_2 because I think you are using v1_1 and in that version there was not author_id in the comments table 🤔

![image](https://github.com/adamilare/railsblog/assets/93039359/9b315068-4cc2-40c3-b997-280d5bf7a53b)

Reply:
Thank you for clearing that out. I wonder where that version conflicts emanated from. I found it changed checking it out now.